aya/src/bpf.rs

```diff
@@ -876,6 +912,10 @@ pub enum BpfError {
     #[error("program error: {0}")]
     /// A program error
     ProgramError(#[from] ProgramError),
+
+    /// Unsupported Map type
+    #[error("Unsupported map types found")]
+    UnsupportedMap,
```

Comment:
This should be `MapError::Unsupported { map_type: u32 }`

Reply:
Done